src/python/datastore/locks.py

```diff
@@ -144,11 +119,6 @@ def acquire_lock(key_name,
           retries=TRANSACTION_RETRIES)
 
       if lock_entity.holder == bot_name:
-        _update_lock_statistics(
```

Comment:
These are ton of unnecessary stats that are piling up in DB, so decided to remove.